app/policies/event_policy.rb

```diff
@@ -2,24 +2,29 @@
 
 # Rules governing permissions for Events
 class EventPolicy < ApplicationPolicy
+  # Everyone can view Event details
```

Comment:
Are these comments really needed? You can pretty clearly see that this is an EventPolicy and users are what policies get checked for

Reply:
If I ever decide to add RDoc, these comments will show up on the policies, which is pretty nifty!